packages/generator-single-spa/src/vue/generator-single-spa-vue.js

```diff
@@ -2,8 +2,30 @@ const Generator = require("yeoman-generator");
 const { spawnSync } = require("child_process");
 const util = require("util");
 const commandExists = util.promisify(require("command-exists"));
+const chalk = require("chalk");
 
 module.exports = class SingleSpaVueGenerator extends Generator {
+  constructor(args, opts) {
+    super(args, opts);
+
+    this.option("packageManager", {
```

Comment:
If we do not provide this to Vue CLI as an argument, it will prompt the user for this. Why do we need to do it here?

Is it so that we can provide the correct final instructions for the instant test (`yarn serve` versus `npm run server`)?

Reply:
I'm not getting prompted for which package manager to use with \@vue/cli 4.4.1, but I'm realizing that its because at some point it saved it into my `~/.vuerc`. And yes, it is helpful to have that option before trigging the vue cli so we can show the corresponding command once it completes.

Reply:
I generally would lean towards letting Vue CLI handle as many options as possible. That way there is less for create-single-spa to do, and also the UI we present to the developer is familiar to them if they have used Vue CLI.

Perhaps an alternative would be to check for the presence of a `yarn.lock` or `package-lock.json` file after Vue has finished? And then change the colored instructions based on that?

What do you think?

Reply:
Yea, that would work well! I'll change it.